src/resources/transactions.js

```diff
@@ -142,30 +142,37 @@ class Transactions {
   }
 
   async send({ userId, accountId, currency, toAddress, amount } = {}) {
+    if (!toAddress) {
+      throw new SDKError(400, '[Zabo] Missing `toAddress` parameter. See: https://zabo.com/docs#send-a-transaction')
+    } else if (!currency) {
+      throw new SDKError(400, '[Zabo] Missing `currency` parameter. See: https://zabo.com/docs#send-a-transaction')
+    } else if (!amount) {
+      throw new SDKError(400, '[Zabo] Missing `amount` parameter. See: https://zabo.com/docs#send-a-transaction')
+    } else if (!accountId) {
+      throw new SDKError(400, '[Zabo] Missing `accountId` parameter. See: https://zabo.com/docs#send-a-transaction')
+    } else if (!uuidValidate(accountId, 4)) {
+      throw new SDKError(400, '[Zabo] `accountId` must be a valid UUID v4. See: https://zabo.com/docs#send-a-transaction')
+    }
+
+    currency = currency.toUpperCase()
+
     if (utils.isNode()) {
       if (!userId) {
         throw new SDKError(400, '[Zabo] Missing `userId` parameter. See: https://zabo.com/docs#send-a-transaction')
       } else if (!uuidValidate(userId, 4)) {
         throw new SDKError(400, '[Zabo] `userId` must be a valid UUID v4. See: https://zabo.com/docs#send-a-transaction')
-      } else if (!accountId) {
-        throw new SDKError(400, '[Zabo] Missing `accountId` parameter. See: https://zabo.com/docs#send-a-transaction')
-      } else if (!uuidValidate(accountId, 4)) {
-        throw new SDKError(400, '[Zabo] `accountId` must be a valid UUID v4. See: https://zabo.com/docs#send-a-transaction')
-      } else if (!toAddress) {
-        throw new SDKError(400, '[Zabo] Missing `toAddress` parameter. See: https://zabo.com/docs#send-a-transaction')
-      } else if (!amount) {
-        throw new SDKError(400, '[Zabo] Missing `amount` parameter. See: https://zabo.com/docs#send-a-transaction')
       }
 
-      if (currency.toUpperCase() == 'HBAR') {
```

Comment:
Did you just see this as too redundant?

Reply:
I moved this to some lines above to standardize the arguments at once.
However, this is a good catch, lines 157 and 192 are duplicated. I'll push a fix.